src/views/tickets/Screenshot.js

```diff
@@ -74,8 +72,14 @@ export default class Screenshot extends Component {
             />
           </Slide>
         );
-        carouselTemp.push(carouselImg);
-        this.setState({ carousel: carouselTemp });
+
+        this.carousel.push(carouselImg);
+        getArtifact(
+          this.props.ticketID,
+          'recognize-' + filename.slice(0, -4) + '.ocr'
+        ).then(res => {
+          this.ocrText.push(res);
```

Comment:
Since the rest of the code is removing promise-style logic, might be worthwhile to convert this promise chain to `await` for consistency.

Reply:
Sure, also didn't write this just moved it around but I agree, using promise and async together was leading to weird errors at times

Reply:
That makes sense. Thanks :)